task4.js

```diff
@@ -0,0 +1,22 @@
+// Write a function that performs a deep copy of an object (without using JSON).
+
+function deepCopy(obj) {
+  //   structuredClone
+  const copiedObj = structuredClone(obj);
```

Comment:
create separate variable in cases when you are going to reuse that value or if you code is too long e.t.c
but for simple cases you don't need that. Just return correct value
`return  structuredClone(obj);`